src/mapgen/mineclone_stronghold.cpp

```diff
@@ -0,0 +1,82 @@
+#include "noise.h"
+#include <vector3d.h>
+#include "mineclone_stronghold.h"
+#include <math.h>
+#include "constants.h"
+#include "util/numeric.h"
+#include <stdlib.h>
+
+#include <iostream>
+
+#define PI 3.14159265358979323846
```

Comment:
why are you defining pi here? since you are using the math.h library, why cant you use M_PI as seen here: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/math.h.html

Reply:
Seems like a leftover before I included math.h